GameServer/Logic/GameObjects/Champion.cs

```diff
@@ -191,16 +194,19 @@ public int getTeamSize()
 
         public Spell castSpell(byte slot, float x, float y, Unit target, int futureProjNetId, int spellNetId)
         {
-            if (slot == 13)
+            Spell s = null;
+            foreach (Spell t in spells)
             {
-                Logger.LogCoreInfo("Attempted to cast recall. Not implemented");
-                return null;
+                if (t.getSlot() == slot)
+                {
```

Comment:
I don't really approve the way this is done, but I guess it'll do for now, as we'll probably refactor this later anyway